### ISSUE TITLE
--net=host option doesn't work on windows

### ISSUE THREAD

**Issue by Jeducious** [crowdrender/cr-docker#15]
# What's up?
Dockers [documentation ](https://docs.docker.com/network/host/) reveals that the host driver is only available on linux, testing on both linux and windows confirms that using;

```bash
docker run -t \
 --name "Crowdrender-Server" \
 -e token=<login_token> \
 --net=host \
zocker160/blender-crowdrender:latest
```
...doesn't work on windows. What's more is that there's no obvious warning about this in the readme.md file, nor from docker itself when running the above command in windows. So anyone using the instructions from the readme.md would likely not know where to start.

# Good news is
As it turns out, the -p option works, this is described in the readme file, so I think a small modification to the documentation here on GitHub would go a long way since I've proved today you can use docker on windows with the crowdrender images. You just replace the --net=host option with -p 9669-9694:9669-9694 which causes docker to forward those ports to the container.

**Comment by zocker-160** [comment]
This is documented in the readme, just the wording is bad, since it says `docker desktop` which I would expect Windows users to use.
On the other hand, if you are on Windows, using Docker sounds like a bad idea, because you leave performance on the table.

**Comment by Jeducious** [comment]
Closed by #14
